utils/config/examples/daos_server_sockets.yml

```diff
@@ -46,38 +46,41 @@ engines:
   # - CRT_CTX_SHARE_ADDR=1
   # - CRT_CTX_NUM=8
 
-  # Storage definitions
-  scm_mount: /mnt/daos0
+  storage:
+  -
+    # Storage definitions
```

Comment:
In my mind class should be at the top of each storage tier section, the user has to know which parameters to fill out for each class and I think it's easier if the class is given first.